app/models/playlist.rb

```diff
@@ -0,0 +1,11 @@
+class Playlist < ApplicationRecord
+  validates :name, presence: true
+  validate :has_content
+  validates :energy, :danceability, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100 }
+
+  def has_content
```

Comment:
`has_content` n'est pas un bon nom: cela ne traduit pas exactement ce que fait la méthode ici.

Ta méthode vérifie que au moins un critère est remplie, un meilleur nom serait donc:

```suggestion
  def at_least_one_critera_is_filled
```

On a du coup plus haut:

```
validate :at_least_one_critera_is_filled
```

c'est clairement plus lisible :)